### ISSUE TITLE
Multiple others group on pie chart

### ISSUE THREAD

**Issue by paulo-lada** [amcharts/amcharts4#3759]
Hello, is there a way to add multiple "others" group? My data has a lot of little slices and the slices works perfectly, but when I zoom in there's still lots of little slices and sometimes can be hard to identify, so I was trying a way to add another layer others to the graph, can it be done in some way?

**Comment by martynasma** [comment]
No, I'm afraid there's no such built-in functionality. The SliceGrouper plugin only supports single Others group :(
